scripts/sync-agents.sh

```
@@ -0,0 +1,88 @@
+    "obsidian-home-base"
+    "obsidian-image-manager"
+    "obsidian-oxygen-settings"
+    "obsidian-property-over-file-name"
+    "obsidian-seo"
+    "obsidian-ui-tweaker"
+    "obsidian-vault-cms"
+    "obsidian-zenmode"
+)
+
+MASTER_AGENTS_PATH="$DEV_ROOT/$MASTER_REPO/.agents"
+MASTER_AGENTS_MD_PATH="$DEV_ROOT/$MASTER_REPO/AGENTS.md"
```

Shellcheck (0.11.0)

[warning] MASTER_AGENTS_MD_PATH appears unused. Verify use (or export if used externally).

(SC2034)